_posts/2022-02-23-paying-contributors-sponsoring-projects.md

```diff
@@ -0,0 +1,102 @@
+
+Each month, we set aside $1,000 specifically to give to outside contributors for any contributions to ESLint. The TSC reviews outside contributions at the end of each month and awards at least $100 to every outside contributor who has made a non-trivial contribution to ESLint. Once again, these contributions are not limited to coding, but can be anything that positively impacts the project. The contributors don't need to apply ahead of time or ask for permission; if you make a significant contribution, you'll get an email from the TSC letting you know how to collect your payment.
+
+In 2021, we awarded over $6,000 to outside contributors, and we look forward to awarding more in 2022. The bottom line here: if you make any non-trivial contribution to ESLint, you will get paid for it.
+
+### Supporting our dependencies
+
+As we [announced in 2020](https://eslint.org/blog/2020/09/supporting-eslint-dependencies), we actively seek out and donate to our dependencies. In general, any project we directly depend on that has an Open Collective account will receive a donation from ESLint. We are currently supporting five of our dependencies with monthly donations of $150:
+
+* [Ajv](https://npmjs.com/package/ajv) is a JSON schema validator that ESLint uses to validate configuration.
+* [Eleventy](https://www.11ty.dev/) is a Node.js-based static site generator that we use to create [eslint.org](http://eslint.org).
+* [Sindre Sorhus](https://sindresorhus.com/) is a prolific open source developer, and ESLint uses several of his modules, including [`chalk`](https://npmjs.com/package/chalk), [`globals`](https://npmjs.com/package/globals), [`import-fresh`](https://npmjs.com/package/import-fresh), [`strip-ansi`](https://npmjs.com/package/strip-ansi), and [`strip-json-comments`](https://npmjs.com/package/strip-json-comments).
+* [debug](https://npmjs.com/package/debug) is a small tool ESLint uses to output debugging messages when you use the `--debug` flag. 
+* [lint-staged](https://npmjs.com/package/lint-staged) is a simple precommit hook manager that makes it easy to setup linting in your development process.
+
+We think it's important for open source projects that receive a lot of donations to take care of their dependencies. After all, if your project couldn't exist without those dependencies, then they deserve your support.
+
+You can always check to see which projects ESLint is supporting on our [Open Collective page](https://opencollective.com/eslint#category-CONTRIBUTIONS).
+
+### Supporting the community
+
+Once we started supporting our upstream dependencies, we though, what about the downstream dependencies? There are a lot of projects related to ESLint that make the ESLint ecosystem better. Whether that be a different set of rules, a custom parser, or anything that makes ESLint better to use. We are currently donating $150 per month to these projects:
```

Comment:
```suggestion
Once we started supporting our upstream dependencies, we thought, what about the downstream dependencies? There are a lot of projects related to ESLint that enrich the ESLint ecosystem, whether that be a different set of rules, a custom parser, or anything that makes ESLint better to use. We are currently donating $150 per month to these projects:
```

Figured better to avoid two "better"s. :-)